src/ewatercycle/observation/grdc.py

```diff
@@ -43,17 +44,17 @@ def get_grdc_data(
         column: optional. Name of column in dataframe. Default: "streamflow".
 
     Returns:
-        grdc data in a dataframe and metadata.
+        grdc data in a dataframe with metadata in its attrs.
 
     Examples:
         .. code-block:: python
 
             from ewatercycle.observation.grdc import get_grdc_data
 
-            df, meta = get_grdc_data('6335020',
+            ds = get_grdc_data('6335020',
                                     '2000-01-01T00:00Z',
                                     '2001-01-01T00:00Z')
-            df.describe()
+            ds.describe()
```

Comment:
xr.Dataset has no describe method.
```suggestion
            ds.to_dataframe().describe()
```

Reply:
good catch, replaced with print of ds